examples/lib/stories/effects/size_effect_example.dart

```diff
@@ -15,35 +15,29 @@ class SizeEffectExample extends FlameGame with TapDetector {
     down, depending on its current state.
   ''';
 
-  late RectangleComponent square;
+  late Component shape;
   bool grow = true;
 
   @override
   Future<void> onLoad() async {
-    square = RectangleComponent.square(
```

Comment:
I wondered how this one could have worked previously, the answer was simple; it didn't. :sweat_smile: